src/Sarif.Viewer.VisualStudio.Core/SarifCommandPackage.vsct

```diff
@@ -183,5 +212,17 @@
       <IDSymbol name="cmdidIFixedThisCommand" value="0x0309"/>
     </GuidSymbol>
 
+    <GuidSymbol name="guidSarifErrorListResultSourceServiceCommandSet" value="{b04424d9-49bc-4e04-9ecc-ad5b68cce4bc}">
+      <!-- MENUS -->
+      <IDSymbol name="9655cb55-c0b8-4636-9d9a-6a8685f76ee0" value="0x5000"/>
+
+      <!-- GROUPS -->
+      <IDSymbol name="2ee78ada-af4b-4e2c-aaa8-0326ef12d611" value="0x6000"/>
+      <IDSymbol name="9abdbc25-a08d-49dd-9065-4eb669194441" value="0x6001"/>
+
+      <!-- BUTTONS -->
+      <IDSymbol name="c4883543-de09-4989-83c2-16e40d105831" value="0x8B67"/>
```

Comment:
>c4883543 [](http://example.com/codeflow?start=22&length=8)

should we use the meaningful names as other IDSymbols?

Reply:
I did it this way because there could potentially be a lot of these. I actually wrote a tool to generate all of the XML

Reply:
And this is why I created a separate command set.